app/scripts/controllers/client/CreateClientController.js

```diff
@@ -127,6 +192,74 @@
                     this.formData.savingsProductId = null;
                 }
 
+                if(scope.enableAddress===true)
+                {
+                    for(var i=0;i<scope.addressArray.length;i++)
+                    {
+                        var temp=new Object();
+                        if(scope.addressArray[i].addressTypeId)
```

Comment:
If address type id is not enabled and rest of them are enabled in which address category you are going to save this new address?